pmlc/dialect/pxa/analysis/strides.cc

```diff
@@ -213,22 +213,41 @@ StrideRange StrideInfo::range() const {
 }
 
 AffineValueExpr StrideInfo::toValueExpr(MLIRContext *ctx) const {
+  std::map<int64_t, std::pair<mlir::BlockArgument, int64_t>> ordered;
+
+  for (auto kvp : strides) {
+    int64_t nestLevel = 0;
+    auto block = kvp.first.getOwner();
+    assert(block);
+    auto parent = block->getParentOp();
+    while (parent) {
+      nestLevel++;
+      parent = parent->getParentOp();
+    }
+
+    ordered.emplace(
+        nestLevel * 10000 + kvp.first.getArgNumber(),
```

Comment:
10000 is a magic number here, basically it represents a max number of arguments per level.  there may be a better way to do this.